README.md

```diff
@@ -48,3 +48,21 @@ To get started -
 5. Build the API Docs, via `yarn run build`
 6. Ready! Now you can launch the API Docs, via `yarn gitbook serve`
 7. Access the API Docs via [http://localhost:4000](http://localhost:4000)
+
+## tests
+
+Run a Substrate chain node:
+```bash
+curl https://getsubstrate.io -sSf | bash;
```

Comment:
Until getsubstarte actually works as expected, we need to point to master. (This approach is actually what made you have a horror of a time here, I would not like the repeat that with others)